### DISCUSSION TITLE
How can we add custom range type options to native filter?

### DISCUSSION THREAD

**Discussion by almirb** [apache/superset#21215] [Q&A / Help]
Hi peolple!
We are using Superset in our company for some months, and this tool is awesome! Thanks to the Superset community!

At Range Type, we can have "Last" (last day, last week, last month...), "Previous" (previous calendar week, previous calendar month...), custom and advanced.

One thing people always ask to me, is how to filter the dashboards using "current month" preset.
We can achieve that by using "Custom" and setting the date and time manually, but It gives a little more work as the users needed to do It every time they get on the dashboard.

How can I add this kind of preset do the time range filters?  
Thanks!

![image](https://user-images.githubusercontent.com/1752962/186952989-ebe73db5-8f4a-4fb4-9c6d-36f2abcffb69.png)

**Comment by kamalkeshavani-aiinside** [comment]
I think you can try 'Native Filter Set'.
